src/pages/User/components/PullRequests/UserInfo/Message/getMessage.js

```diff
@@ -0,0 +1,44 @@
+import { pullRequestAmount, otherReposAmount } from '../../pullRequestAmount';
+
+const messages = [
+  "It's not too late to start!",
+  'Off to a great start, keep going!',
+  'Off to a great start, keep going!',
```

Comment:
Duplicate messages? Why so?

Reply:
Because on frogtoberfest we have to get 10 Total PR for the challenge completion criteria, so due to limited no of things to be said back to user, i have used some messages again for motivation.

Reply:
No need to repeat the same messages. May be you can change rephrase some of the sentences if that is what you need.